### PR TITLE
Extract cm

### PR DESCRIPTION
**Pull request by Eabramov84**

### CODE REVIEW

File: recipes-aos/aos-servicemanager/files/aos-servicemanager.service [hunk 1]
```diff
@@ -1,7 +1,7 @@
 [Unit]
 Description=AOS Service Manager
-Wants=network-online.target aos-iamanager.service
-After=network-online.target aos-iamanager.service
+Wants=network-online.target
```
Comment (al1img) [inline recipes-aos/aos-servicemanager/files/aos-servicemanager.service:3]: But SM still requires IAM, why not start it after IAM?

File: recipes-aos/aos-updatemanager/files/aos_updatemanager.cfg [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
     "ID": "um",
-    "ServerUrl": "aossm:8091",
+    "ServerUrl": "aoscm:8091",
     "CACert": "/etc/ssl/certs/rootCA.pem",
     "CertStorage": "/var/aos/crypt/um/",
```
Comment (al1img) [inline recipes-aos/aos-updatemanager/files/aos_updatemanager.cfg:5]: Also, add dep in UM service file to start after CM.

File: recipes-aos/aos-iamanager/aos-iamanager_git.bbappend [hunk 3]
```diff
@@ -9,6 +9,8 @@ SRC_URI_append = "\
     file://encrypt.sh \
 "
 
+BRANCH = "update-iam-interface"
```
[the remark this answers is not in the document]
Comment (Eabramov84) [reply recipes-aos/aos-iamanager/aos-iamanager_git.bbappend:12]: added by mistake. Fixed

File: recipes-aos/aos-communicationmanager/aos-communicationmanager_git.bbappend [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+"
+
+inherit systemd
+
+SYSTEMD_SERVICE_${PN} = "aos-communicationmanager.service"
+
+MIGRATION_SCRIPTS_PATH = "/usr/share/communicationmanager/migration"
+
+FILES_${PN} += " \
+    ${sysconfdir}/aos/aos_communicationmanager.cfg \
```
[the remark this answers is not in the document]
Comment (Eabramov84) [reply recipes-aos/aos-communicationmanager/aos-communicationmanager_git.bbappend:15]: done

File: recipes-aos/aos-updatemanager/files/aos_updatemanager.cfg (hunk 2 above)
Comment (Eabramov84) [reply recipes-aos/aos-updatemanager/files/aos_updatemanager.cfg:5]: added

File: recipes-core/images/aos-image-minimal.inc [hunk 5]
```diff
@@ -5,4 +5,5 @@ BOARD_MODEL ?= "vm-dev;1.0"
 BUNDLE_IMAGE_VERSION ?= "1.0"
 
 ROOTFS_IMAGE_VERSION ?= "${BUNDLE_IMAGE_VERSION}"
-BOOT_IMAGE_VERSION ?= "${BUNDLE_IMAGE_VERSION}"
+ROOTFS_REF_VERSION ?= "${BUNDLE_IMAGE_VERSION}"
+BOOT_IMAGE_VERSION ?= "${BUNDLE_IMAGE_VERSION}"
```
Comment (al1img) [inline recipes-core/images/aos-image-minimal.inc:9]: Add new line at the end.

File: recipes-aos/aos-communicationmanager/files/aos_communicationmanager.cfg [hunk 6]
```diff
@@ -0,0 +1,38 @@
...
+                "priority": 0,
+                "isLocal": true
+            }
+        ]
+    },
+    "migration": {
+        "migrationPath": "/usr/share/communicationmanager/migration",
+        "mergedMigrationPath": "/var/aos/communicationmanager/migration"
+    }
+}
```
Comment (al1img) [inline recipes-aos/aos-communicationmanager/files/aos_communicationmanager.cfg:38]: Add new line at the end.

File: recipes-aos/aos-iamanager/files/aos_iamanager.cfg [hunk 7]
```diff
@@ -92,6 +92,22 @@
...
+            ],
+            "AlternativeNames": [
+                "aoscm"
+            ],
+            "Params": {
+                "StoragePath": "/var/aos/crypt/cm/"
+            }
         }
     ]
 }
```
Comment (al1img) [inline recipes-aos/aos-iamanager/files/aos_iamanager.cfg:113]: Add new line at the end,

**Review by al1img — APPROVED** [review]

**Pull request merged**